nservicebus/service-fabric/sagas.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Service Fabric Persistence Sagas
+reviewed: 2017-03-22
+component: ServiceFabricPersistence
+---
+
+## Reliable collections
+
+When using the Service Fabric Persistence with a reliable service, saga data is stored in reliable dictionaries.  
+
+## Saga data serialization
+
+Service Fabric Persistence stores saga data in JSON format using uses Json.NET. 
+
+Saga data serialization can be configured by providing custom [JsonSerializerSettings](http://www.newtonsoft.com/json/help/html/T_Newtonsoft_Json_JsonSerializerSettings.htm) instance 
+
+snippet: ServiceFabricPersistenceSagaJsonSerializerSettings
+
+custom [JsonReader](http://www.newtonsoft.com/json/help/html/T_Newtonsoft_Json_JsonReader.htm) instance
+
+snippet: ServiceFabricPersistenceSagaReaderCreator
+
+or custom [JsonWriter](http://www.newtonsoft.com/json/help/html/T_Newtonsoft_Json_JsonWriter.htm) instance
+
+snippet: ServiceFabricPersistenceSagaWriterCreator
+
+## Saga data storage 
+
+By default saga data stored in multiple reliable collections - one per saga data type. Reliable collection name can be changed at the level of saga data type using `ServiceFabricSagaAttribute.CollectionName`
```

Comment:
drop the "ServiceFabricSagaAttribute.CollectionName" part of this sentence will make it more resilient to change. u already have the snippet below anyway